Profile/Profile/Presentation/Profile/Subviews/ProfileSupportInfo.swift

```diff
@@ -0,0 +1,187 @@
+//
+//  ProfileSupportInfo.swift
+//  Profile
+//
+//  Created by Eugene Yatsenko on 13.12.2023.
+//
+
+import SwiftUI
+import Theme
+import Core
+
+struct ProfileSupportInfo: View {
+
+    struct LinkViewModel {
+        let url: URL
+        let title: String
+    }
+
+    @ObservedObject var viewModel: ProfileViewModel
+
+    var body: some View {
+        Text(ProfileLocalization.supportInfo)
+            .padding(.horizontal, 24)
+            .font(Theme.Fonts.labelLarge)
+        VStack(alignment: .leading, spacing: 24) {
+            viewModel.contactSupport().map(supportInfo)
+            viewModel.config.agreement.tosURL.map(terms)
+            viewModel.config.agreement.privacyPolicyURL.map(privacy)
+            viewModel.config.agreement.cookiePolicyURL.map(cookiePolicy)
+            viewModel.config.agreement.dataSellContentURL.map(dataSellContent)
+            viewModel.config.faq.map(faq)
+            version
+        }
+        .cardStyle(
+            bgColor: Theme.Colors.textInputUnfocusedBackground,
+            strokeColor: .clear
+        )
+    }
+
+    @ViewBuilder
+    private func supportInfo(url: URL) -> some View {
+        button(
+            linkViewModel: .init(
+                url: url,
+                title: ProfileLocalization.contact
+            ),
+            isEmailSupport: true
+        )
+
+    }
+
+    private func terms(url: URL) -> some View {
+        navigationLink(
+            viewModel: .init(
+                url: url,
+                title: ProfileLocalization.terms
+            )
+        )
+    }
+
+    private func privacy(url: URL) -> some View {
+        navigationLink(
+            viewModel: .init(
+                url: url,
+                title: ProfileLocalization.privacy
+            )
+        )
+    }
+
+    private func cookiePolicy(url: URL) -> some View {
+        navigationLink(
+            viewModel: .init(
+                url: url,
+                title: ProfileLocalization.cookiePolicy
+            )
+        )
+    }
+
+    private func dataSellContent(url: URL) -> some View {
+        navigationLink(
+            viewModel: .init(
+                url: url,
+                title: ProfileLocalization.doNotSellInformation
+            )
+        )
+    }
+
+    private func faq(url: URL) -> some View {
```

Comment:
I guess `supportInfo(....` and `faq( .... `can also use a common named method. Thoughts?